WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsInterface.java

```diff
@@ -1051,9 +1052,17 @@ private void loadCachedSettings() {
                     mContext.getString(R.string.file_size_in_megabytes),
                     mContext.getString(R.string.file_size_in_gigabytes),
                     mContext.getString(R.string.file_size_in_terabytes) };
-            String spaceAllowed = FormatUtils.formatFileSize(mSite.getSpaceAllowed(), units);
-            String quotaAvailableSentence = String.format(mContext.getString(R.string.site_settings_quota_space_value),
-                    percentage, spaceAllowed);
+
+            String quotaAvailableSentence;
+            if (mSite.getPlanId() == PlansConstants.BUSINESS_PLAN_ID) {
```

Comment:
Do we also need to consider `JETPACK_BUSINESS_PLAN_ID` here?

Reply:
Good question @nbradbury I checked this morning with @elibud and jetpack plans only count for CDN unlimited storage / backup storage, but the actual site content storage is not something jetpack would cover (i.e. a jetpack connected site is self hosted so storage is up to whoever is giving the customer such a service)